homeassistant/components/homeassistant/__init__.py

```diff
@@ -145,24 +151,24 @@ async def async_handle_turn_service(service: ha.ServiceCall) -> None:
             await asyncio.gather(*tasks)
 
     hass.services.async_register(
-        ha.DOMAIN, SERVICE_SAVE_PERSISTENT_STATES, async_save_persistent_states
+        DOMAIN, SERVICE_SAVE_PERSISTENT_STATES, async_save_persistent_states
```

Comment:
This is the main change... the DOMAIN used when registering services now comes from local `const.py` not from `core`